app/main.py

```diff
@@ -1,8 +1,51 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: str,
+                 brand: str
+                 ) -> None:
+
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: int,
+                 clean_power: int, average_rating: int,
+                 count_of_ratings: int
+                 ) -> None:
+
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> None:
+        self.cars = cars
+
+        income = 0
+        for i in cars:
+            if i.clean_mark < self.clean_power:
+                income += self.calculate_washing_price(i)
+                self.wash_single_car(i)
+
+        return round(income, 1)
+
+    def calculate_washing_price(self, car: Car) -> None:
+        cost = (car.comfort_class * (self.clean_power - car.clean_mark)
+                * self.average_rating) / self.distance_from_city_center
+
+        return round(cost, 1)
+
+    def wash_single_car(self, car: Car) -> None:
+        if self.clean_power >= car.clean_mark:
+            car.clean_mark = self.clean_power
+
+    def rate_service(self, single_rate: int) -> None:
+        self.average_rating = round(
+            (self.average_rating * self.count_of_ratings + single_rate)
+            / (self.count_of_ratings + 1), 1)
```

Comment:
Write all calculations to a separate variable and then assign a rounded value to the `self.average_rating`. It will help to avoid inconsistent line breaking for `round()` in your code.